build/stage-build-security-ios.yml

```diff
@@ -0,0 +1,19 @@
+parameters:
```

Comment:
The stages files are almost the same between Android and iOS. Could they be combined?
The other stages that we already have are not split per platform. The split is done at the jobs/steps level. I think we could have the same thing in this case too.